lua/entities/gmod_wire_expression2/base/preprocessor.lua

```diff
@@ -108,7 +108,7 @@ function PreProcessor:RemoveComments(line)
 					self.blockcomment = true -- We're now inside a block comment
 				elseif type == "normal" then -- We found a # instead
 					local pos = comments[i].pos
-					local incPos, incEndPos = line:sub(pos):find("#include%s*%b\"\"")
+					local incPos, incEndPos = line:sub(pos):find("#include%s+%b\"\"")
```

Comment:
Should have changed "%s+" to " +". A space, that is no other type of whitespace.

Reply:
Tabs

Reply:
Wasn't previously allowed, you want to make this happen?
I'm cool with it. Just note what changes you're introducing.

Also, there should be an error when there's no path string on the same line then.

Reply:
No